ctapipe/configuration/configuration.py

```diff
@@ -0,0 +1,312 @@
+            with open('example.ini', 'w') as configfile: 
+                configParser.write(configfile)
+
+                 #with open( self.statusfile, "w") as ini_out:
+                 #    self.myConfig.write( ini_out )
+
+                #self._configParser.write(configfile, space_around_delimiters)
+        else:
+            print("Type:",type,'not allowed',file=sys.stderr)
+        
+        
+        
+    #--------------------------------------------------------------------------#
+    def _writeFits(self, filename,implementation=DATAIMPL):
+        """Write an FITS file representation of the configuration state.
+        """
+        if implementation != self.DATAIMPL and implementation != self.HEADERIMPL:
+            print("Implementation :",implementation,'not allowed',file=sys.stderr)
+            return 
+                
+        # hduList will contain one TableHDU per section
+        hduList = fits.HDUList()   
+        
+        # get all Configuration entries
+
+        # loop over section
+        for section in self._entries.keys():
+            if implementation == self.DATAIMPL:
+                # prepare 3 array
+                key_array = []
+                value_array = []
+                comment_array = []
+                # loop over section entries and fill arrays
+                for key,value_comment in self._entries[section].items():
+                    key_array.append(key)
+                    value_array.append(value_comment[self.VALUE_INDEX])
+                    comment_array.append(value_comment[self.COMMENT_INDEX])
+                #create fits.Column form filled arrays
+                ckey = fits.Column(name='key', format='A256', array=np.array(key_array))
+                cvalue = fits.Column(name='value', format='A256', array=np.array(value_array))
+                ccomment = fits.Column(name='comments', format='A256', array=np.array(comment_array))
+                # Create the table
+                #hdu = fits.new_table([ckey, cvalue], tbtype='TableHDU')
+                hdu = fits.TableHDU.from_columns([ckey, cvalue,ccomment])
+                hdu.name=section
+                # append table to hduList
+                hduList.append(hdu)
+        
+        
+            elif (implementation == self.HEADERIMPL):
+                header = fits.Header()
+                for key,value_comments in self._entries[section].items():
+                    header[key]=(value_comments[self.VALUE_INDEX] , value_comments[self.COMMENT_INDEX])
+
+                table_0=fits.TableHDU(data=None, header=header, name=section)
+                hduList.append(table_0)
+        
+
+        hduList.writeto(filename,clobber=True)
+        
+                
+        #table_0.append(filename, data=data, header=header, checksum=False, verify=True)
+
+    
+    #--------------------------------------------------------------------------#
+    def read(self, filenames, type=FITS, implementation=DATAIMPL, encoding=None):
```

Comment:
`type` is a Python built-in keyword and re-setting it to something else can cause big confusion.
Maybe use `format` instead?